ui/pages/permissions-connect/snaps/snap-result/snap-result.js

```diff
@@ -74,22 +83,43 @@ export default function SnapResult({
             paddingTop={2}
             paddingBottom={2}
           >
+            <AvatarIcon
+              className="snap-result__header__icon"
+              iconName={IconName.Confirmation}
+              size={IconSize.Xl}
+              iconProps={{
+                size: IconSize.Xl,
+              }}
+              color={IconColor.successDefault}
+              backgroundColor={BackgroundColor.successMuted}
+            />
             <Text
               fontWeight={FONT_WEIGHT.BOLD}
               variant={TextVariant.headingLg}
               paddingBottom={2}
+              marginTop={4}
             >
               {t('snapResultSuccess')}
             </Text>
-            <Text textAlign={TEXT_ALIGN.CENTER}>
-              {t('snapResultSuccessDescription', [<b key="1">{snapName}</b>])}
+            <Text textAlign={TextAlign.Center}>
+              {t('snapResultSuccessDescription', [
+                <Text as={ValidTag.Span} key="1" fontWeight={FontWeight.Medium}>
+                  {snapName}
+                </Text>,
+              ])}
             </Text>
           </Box>
         )}
         {hasError && (
           <InstallError
             error={requestState.error}
-            title={t('snapResultError')}
+            title={t('snapInstallationErrorTitle')}
```

Comment:
We cannot assume that `SnapResult` is only used for snap install, it is also used for snap update, so we need to change this copy dynamically based on the previous flow. Should be available as `requestState.type`

Reply:
Added changes which will leave update flow untouched. So, only the install flow will be affected. I think update flow is supposed to be updated in another ticket?

Reply:
I have made other changes to the update flow in another PR, but not to the result screens. Do you mind including the updated result screens for updates in this PR as well? Should mostly be copy given that it leverages most of what you've changed already.

Reply:
Added new messages and design for update failed and success screens.